libs/ledger-live-common/src/api/Tron.ts

```diff
@@ -119,7 +119,7 @@ export const createTronTransaction = async (
   if (tokenType === "trc20" && tokenId) {
     const txData: SmartContractTransactionData = {
       function_selector: "transfer(address,uint256)",
-      fee_limit: 10000000,
+      fee_limit: 20000000,
```

Comment:
why this value ?

Reply:
It was discuss by Fabrice https://ledger.slack.com/archives/CRD18A0H5/p1670237662561249?thread_ts=1670221369.653679&cid=CRD18A0H5

We want to upgrade the fee_limit allowing user to be able to do a transaction even tho they do not have enough Energy.